### PR TITLE
GW-24: Upcoming event and rantangan

### PR DESCRIPTION
**Pull request by Julian-UBC**
Created a layout for the upcoming event and rantangan in the home page
- Ignore the button for now --> I will fixed them on the GW-23 ticket
- The build preview: https://deploy-preview-13--gisaubcv3.netlify.app/

### CODE REVIEW

File: src/pages/home/components/events/Events.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { EventLayout } from "shared/layout/EventLayout";
+import { UPCOMINGEVENTS, RANTANGAN } from "./events_data";
+
+export const Events = () => {
+  return (
+    <div className="flex justify-between flex-wrap">
+      <EventLayout
```
Comment (chrishadrian) [inline src/pages/home/components/events/Events.js:9]: I think it's better to use `constants.map((constant) => { <EventLayout />}`. So you also need to change the data file to a list of objects.

File: src/pages/home/components/events/events_data.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// import RumahHantu from "../../../../assets/events/rumah_hantu.jpg";
+
+/**
+ * events field:
+ * - desc
+ * - img
+ * - link
+ */
+
+export const UPCOMINGEVENTS = {
```
Comment (chrishadrian) [inline src/pages/home/components/events/events_data.js:10]: If you decided to use the map thing that I mentioned above, just do `export const EVENTS = [UPCOMINGEVENTS, RANTANGAN]`

**Review by chrishadrian — APPROVED** [review]

File: src/pages/home/components/events/events_data.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+// import RumahHantu from "../../../../assets/events/rumah_hantu.jpg";
```
[the remark this answers is not in the document]
Comment (Julian-UBC) [reply src/pages/home/components/events/events_data.js:1]: Is there any reason for that?

File: src/pages/home/components/events/Events.js (hunk 1 above)
Comment (Julian-UBC) [reply src/pages/home/components/events/Events.js:9]: Ahh I see where you are coming from. I will take this into consideration when I work on the responsive ticket. I'll see which one is a better approach.

File: src/pages/home/components/events/events_data.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (chrishadrian) [reply src/pages/home/components/events/events_data.js:1]: It's just the best practice of react from what I know

[the remark this answers is not in the document]
Comment (Julian-UBC) [reply src/pages/home/components/events/events_data.js:1]: Never heard of it before

[the remark this answers is not in the document]
Comment (Julian-UBC) [reply src/pages/home/components/events/events_data.js:1]: Since a lot of files are already named `constants.js`, we'll just stick to that then.

**Review by jenaarianto — APPROVED** [review]

**Pull request merged**